experiments/AtmosLES/risingbubble.jl

```diff
@@ -64,64 +154,108 @@ function init_risingbubble!(bl, state, aux, (x, y, z), t)
     e_kin = FT(0)
     e_pot = gravitational_potential(bl.orientation, aux)
     ρe_tot = ρ * total_energy(e_kin, e_pot, ts)
-    state.ρ = ρ
-    state.ρu = ρu
-    state.ρe = ρe_tot
-    state.moisture.ρq_tot = ρ * q_pt.tot
+
     ρχ = FT(0)
+
+    # We inject tracers at the initial condition at some specified z coordinates
     if 500 < z <= 550
         ρχ += FT(0.05)
     end
+
+    # We want 4 tracers
     ntracers = 4
+
+    # Define 4 tracers, (arbitrary scaling for this demo problem)
     ρχ = SVector{ntracers, FT}(ρχ, ρχ / 2, ρχ / 3, ρχ / 4)
+
+    # Assign State Variables
+    state.ρ = ρ
+    state.ρu = ρu
+    state.ρe = ρe_tot
+    state.moisture.ρq_tot = ρ * q_pt.tot
     state.tracers.ρχ = ρχ
 end
 
+# ## Model Configuration
+# We define a configuration function to assist in prescribing the physical model. The purpose of this is to populate the [`CLIMA.AtmosLESConfiguration`](../Atmos/Model/AtmosModel.md) with arguments appropriate to the problem being considered. 
+"""
+    config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
+
+Arguments
+- FT = Floating-point type. Currently `Float64` or `Float32`
+- N  = DG Polynomial order
+- resolution = 3-component Tuple (Δx, Δy, Δz) with effective resolutions for Cartesian directions
+- xmax, ymax, zmax = Domain maximum extents. Assumes (0,0,0) to be the domain minimum extents unless otherwise specified. 
+
+Returns 
+- `config` = Object using the constructor for the `AtmosLESConfiguration`
+"""
 function config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
 
-    # Choose explicit solver
+    # Choose an Explicit Multi-rate Solver from the existing ODESolver options
+    # Apply the outer constructor to define the `ode_solver`. Here
+    # `AtmosAcousticGravityLinearModel` splits the acoustic-gravity wave components
+    # from the advection-diffusion dynamics. The 1D-IMEX method is less appropriate for the problem given the current mesh aspect ratio (1:1)
     ode_solver = CLIMA.MultirateSolverType(
         linear_model = AtmosAcousticGravityLinearModel,
         slow_method = LSRK144NiegemannDiehlBusch,
         fast_method = LSRK144NiegemannDiehlBusch,
         timestep_ratio = 10,
     )
 
-    # Set up three tracers with integer diffusivity scaling
+    # Since we want four tracers, we specify this and include
+    # the appropriate diffusivity scaling coefficients (normally these
+    # would be physically informed but for this demonstration we use
+    # integers corresponding to the tracer index identifier)
     ntracers = 4
     δ_χ = SVector{ntracers, FT}(1, 2, 3, 4)
 
-    # Set up the model
+    # A model coefficient for the turbulence closure is defined,
     C_smag = FT(0.23)
```

Comment:
Can we grab/overload this from CLIMAParameters.jl? It's in [CLIMAParameters.Atmos.SubgridScale](https://github.com/climate-machine/CLIMAParameters.jl/blob/master/src/AtmosParameters.jl#L4). Need to add
```
using CLIMAParameters.Atmos.SubgridScale: C_smag
```

Reply:
There are other experiments which would need this change as well. We can sort those in a separate PR.